src/nodes/remapDataNode.js

```diff
@@ -71,6 +71,37 @@ class RemapDataNode extends Nodes.SystemTaskNode {
     return RemapDataNode.validate(spec, schema);
   }
 
+  static remapData(data, dictionary) {
+    let messages = [];
+    let remapped_data = data.map((item) => {
+      const remapped_item = {};
+      for (const [key, value] of Object.entries(dictionary)) {
+        if (typeof value === "object") {
+          let nested_remapped_data = RemapDataNode.remapData([item], dictionary[key]);
```

Comment:
essa solução resolve recursividade? Ela trata uma quantidade infinita de aninhamentos?
o teste cobre user.name, mas e se fosse user.profile.name?

Reply:
Trata sim, o teste que cobre é o `"should work with nested object on dictionary"` usando o `nodeExampleNestedObject`